client/bindings/java/iota-client-java/lib/src/test/java/org/iota/HighLevelApiTest.java

```diff
@@ -57,13 +55,30 @@ public void testRetryBlock() {
     }
 
     @Test
-    public void testRetryUntilIncludedBlock() throws ClientException {
+    public void testRetryUntilIncludedBlock() throws ClientException, InterruptedException {
         SecretManager secretManager = new MnemonicSecretManager(DEFAULT_DEVELOPMENT_MNEMONIC);
         String[] addresses = client.generateAddresses(secretManager, new GenerateAddressesOptions().withRange(new Range(0, 2)));
+
+        System.out.println(1);
```

Comment:
nice debug :D

Reply:
will keep this comment open for now until the issue is fixed :-)